src/main/scala/com/typesafe/sbt/site/SitePreviewPlugin.scala

```diff
@@ -18,10 +20,15 @@ object SitePreviewPlugin extends AutoPlugin {
 
   //@TODO Add configuration to make server just local
```

Comment:
Since the code now uses `unfiltered.jetty.Server.local` I think this TODO can be removed.